tests/playbooks/k8s_auth.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Log onto OpenShift, do something, then log out
+  hosts: localhost
+  connection: local
+  vars:
+    k8s_developer_password: zupa.8
+  tasks:
+  - block:
+    # It's good practice to store login credentials in a secure vault and not
+    # directly in playbooks.
+    #- include_vars: k8s_passwords.yml
+
+    - name: Log in (obtain auth token)
+      k8s_auth:
+        host: https://192.168.42.55:8443
+#       ssl_ca_cert: ca.pem
+        verify_ssl: no
```

Comment:
This one is confusing. We default `verify_ssl` to `True`. Why we are using `no`?

Reply:
Because we want this to fail by default and if the operator wants it to work, he needs to take the time to make it work. Either by providing the cert chain or disabling it. Either way the point is that the default behavior should be secure to the extent possible.